docs/user/rstudio/ide/guide/documents/visual-editor.qmd

```diff
@@ -44,20 +39,32 @@ See the [editing shortcuts](https://quarto.org/docs/visual-editor/options.html#s
 
```

Comment:
Keyboard shortcuts in this doc are mac only. May want to add windows as well.

Reply:
Added, thanks!